Assets/Scripts/Player/PlayerGhostRun.cs

```diff
@@ -82,8 +85,12 @@ private void UpdateGhost()
             currentDataIndex++;
             ghostRunnerTimer = 0;
         }
-    }
 
+        if (!OptionsPreferencesManager.GetGhostToggle())

```

Comment:
honestly not a bad way to do it

Reply:
maybe we should pull all of the values into a static class that's written to player prefs when the game closes, that way we dont have to pull from files at runtime in update